server/db/data/recipedia.schema

```diff
@@ -75,4 +74,4 @@ folate: float .
 cname: string .
 
 contains: [uid] @reverse .
-categorized_as: uid .
+categorized_as: [uid] .
```

Comment:
Made it a list of uids so that an ingredient can be categorized into multiple categories.